Assets/Styles/trainingdemo-filtered.scss

```diff
@@ -1,10 +1,10 @@
 @keyframes fadeIn {
     to {
-        opacity: 1;
+        opacity: 100%;

```

Comment:
Miért a százalék? Eddig még sose láttam ilyet opacitynél.

Csak akkor lesz ok ha az scss transpilert be lehet állítani, hogy a css kimenetben alakítsa vissza alfa értékre. (De akkor már tényleg értelmetlennek tűnik.) Mert a százalékos értéknek [elég rossz a böngésző támogatása](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#browser_compatibility), különösen. Illetve macen ugye minden böngésző valójában Safari reskin szóval ott teljesen kiesik.

Reply:
![image](https://user-images.githubusercontent.com/1976647/139608736-9f57b568-03c1-4391-a508-9c335ce3c596.png)

[Lásd](https://stylelint.io/user-guide/rules/list/alpha-value-notation/). A 0% lentebb is ezért van. De jogos, ezt átállítom.